tools/src/ABIChecker/AssemblyChecker.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Checker_ABI
+{
+    internal class AssemblyChecker
+    {
+        private const BindingFlags s_PublicOnlyFlags = BindingFlags.Public | BindingFlags.Instance | BindingFlags.DeclaredOnly | BindingFlags.Static;
```

Comment:
Do not use s_* prefix for private const variable. It should be just "PublicOnlyFlags".